apps/web/package.json

```diff
@@ -65,6 +65,7 @@
     "remark-breaks": "^4.0.0",
     "remark-linkify-regex": "^1.2.1",
     "shared-zustand": "^2.0.0",
+    "solid": "link:@heroicons/react/20/solid",
```

Comment:
ohh no what is this? 😅

Reply:
Do you want me to replace them with plain svg tags? They are the icons that are used in the select and selected option
![image](https://github.com/heyxyz/hey/assets/72187009/da97b95d-a70b-465f-a4c2-13446cf17dd5)
![image](https://github.com/heyxyz/hey/assets/72187009/2969224e-59d4-448a-a054-20c3fc3beed6)

Reply:
we already have heroicon installed, you can use it!

Reply:
done